libs/model/src/test/mockTypes.ts

```diff
@@ -0,0 +1,382 @@
+import {
```

Comment:
we already have schemas in libs/core/schemas... should we move this?

Reply:
This file needs access to both the DB models (model package) and the schemas (core), so I think it needs to stay in model.

Reply:
you can start importing schemas from libs/core to avoid duplication... and relocate from here in case the schema is still missing in libs/core...